etc/cmake/test_helpers.cmake

```diff
@@ -53,6 +53,8 @@ function(add_legacy_test FOLDER NAME)
     )
     set_property(TEST ${NAME} PROPERTY SKIP_RETURN_CODE 77)
   endif()
+  SET(IGRAPH_LIBDIR $<TARGET_FILE_DIR:igraph>)
```

Comment:
Please add a comment here explaining why these two lines are needed. I remember now as we've discussed it only a few weeks ago, but I will definitely forget this in a few months :)

Also, maybe it's worth putting this in an `IF(WIN32)` clause? We are messing around with the system path here, so it's probably worth limiting its impact as much as possible (and we don't need this on Linux or macOS).